docs/source/WritingTests.rst

```diff
@@ -538,6 +538,11 @@ Detailing the ``fetch_asset()`` attributes:
   file, including the file name. The first success will skip the next
   locations. Notice that for ``file://`` we just create a symbolic link in the
   cache directory, pointing to the file original location.
+* ``expire:`` (optional) time period that the cached file will be considered
+  valid. After that period, the file will be dowloaded again. The value can
+  be an integer greater than 0 or a string containig the time and the unit.
+  Example: 10d (ten days). Valid units are s (second), m (minute), h (hour)
+  and d (day).
```

Comment:
I'm not sure about the result. So are you going to adjust the documentation? Someone might want to download the item always and he could use 0 (or -1).

... btw 0 does not mean always expire as ctime is only integer so if you create the file in x.0 sharp, you have 1/1000000s period where it won't be updated... :wink:.